src/buehlmann/buehlmann_model.rs

```diff
@@ -13,6 +17,8 @@ impl DecoModel for BuehlmannModel {
         let mut model = BuehlmannModel {
             compartments: vec![],
             depth: 0.,
+            time: 0,
+            gas: Gas::new(0.21, 0.),
```

Comment:
A little bit misterious instantiation, is it possible to intrduce some named constants? :D

Reply:
right - I planning on exporting some common / standard gasses to be used both internally and as part of common module, that will surely by the a use case for that. Good point though, I'll make it a variable at least for now so that the intention is clear here